Wikipedia/Localizations/ar.lproj/Localizable.strings

```diff
@@ -375,7 +375,6 @@
 "wikitext-upload-result-unknown" = "غير قادر على تحديد نتيجة تحميل النص";
 "wikitext-upload-save" = "جارٍ الحفظ...";
 "wikitext-upload-save-anonymously-warning" = "سوف تحفظ التعديلات تحت اسم عنوان الأيبي الخاص بجهازك. لو $1 سوف يتوفر لك المزيد من الخصوصية.";
-"cc-by-sa-3.0" = "CC BY-SA 3.0";
```

Comment:
Hmm given they contain abbreviations and what not (and aren't we linking to EN url?) should license names even be localized? May want to ping localization folks to see what other projects do...